File: src/main/java/com/cartobucket/auth/routes/mappers/TemplateRequestFilterMapper.java
package com.cartobucket.auth.routes.mappers;

import com.cartobucket.auth.model.generated.TemplateRequestFilter;
import com.cartobucket.auth.model.generated.UserRequestFilter;

import java.util.List;
import java.util.UUID;

public class TemplateRequestFilterMapper {
    public static TemplateRequestFilter to(List<UUID> authorizationServerIds) {
        var filter = new TemplateRequestFilter();
        filter.setAuthorizationServerIds(authorizationServerIds);
        return filter;
    }
}

File: auth-api/src/main/kotlin/com/cartobucket/auth/api/server/routes/mappers/TemplateRequestFilterMapper.kt
// (C)2024
package com.cartobucket.auth.api.server.routes.mappers

import com.cartobucket.auth.api.dto.TemplateRequestFilter
import java.util.UUID

object TemplateRequestFilterMapper {
    fun to(authorizationServerIds: List<UUID>): TemplateRequestFilter =
        TemplateRequestFilter(
            authorizationServerIds = authorizationServerIds,
        )
}
